src/main/java/frc/robot/RobotContainer.java

```diff
@@ -234,27 +235,19 @@ private void configureButtonBindings() {
     shooter.setDefaultCommand(shooter.manualShoot(0));
     arm.setDefaultCommand(arm.hold());
     climber.setDefaultCommand(climber.raiseOrLower(0));
-    driverController.b().onTrue(climber.setRatchet(false));
-    driverController.x().onTrue(climber.setRatchet(true));
 
-    // operator controller
-    operatorController
-        .y()
-        .whileTrue(
-            intake.intake().alongWith(indexer.setPercent(IndexerConstants.INDEX_SPEED.get())));
-
-    operatorController.leftBumper().whileTrue(orchestrator.intakeBasic());
-    operatorController.leftBumper().onFalse(orchestrator.pullBack());
 
+    // operator controller
+    operatorController.a().whileTrue(shooter.manualShoot(ShooterConstants.SHOOT_SPEED));
```

Comment:
I think it'd be helpful to add a short comment to every binding here to name what each of these is, e.g.:
```
 // Hold A: Shoot
 ...
 // Hold B: Expel, in case note is stuck
...
 // Press X: Stow the arm in its home position
...
```

Reply:
agreed